plugins/modules/repository_set.py

```diff
@@ -23,9 +23,9 @@
 ---
 module: repository_set
 version_added: 1.0.0
-short_description: Enable/disable Repositories in Repository Sets
+short_description: Enable/disable Red Hat Repositories
 description:
-  - Enable/disable repositories in repository sets
+  - Enable/disable Red Hat Repositories
```

Comment:
Maybe this should refer to the subscription manifest mechanics.

Reply:
the manifest is a file, I'd not mention it here. "subscription" is a good keyword tho, so maybe:

```
Enable/disable Red Hat Repositories available through subscriptions
```